uwblueprint/community-fridge-kw
# feat: Disable save button before editing

## Brief description. What is this change? 
- If user has not made any changes when editing My Account page, disable the Save button.


### [Disable save changes](https://www.notion.so/uwblueprintexecs/Disable-Save-changes-button-on-My-Account-edit-mode-04def83df8514d86a62337c93c02cdcf)


## Implementation description. How did you make this change?

* Added touch state to keep track whether user had made a change



## Steps to test
1. On My Account page, click 'Edit'.
2. Save button should be disabled.
3. Clicking 'X' should display Cancel modal.
4. Click a field and make changes.
5. The Save button should be enabled.


## Checklist
